samples/csharp_dotnetcore/12.nlp-with-luis/README.md

```diff
@@ -56,7 +67,7 @@ their bots on localhost or running remotely through a tunnel.
 ## Connect to bot using Bot Framework Emulator **V4**
 - Launch the Bot Framework Emulator
 - File -> Open bot and navigate to `botbuilder-samples/samples/csharp_dotnetcore/12.nlp-with-luis` folder
-- Select BotConfiguration.bot file
+- Select `BotConfiguration.bot` file
 # Deploy this bot to Azure
```

Comment:
Update this section with the corrections implemented in other samples, like in [sample 08](https://github.com/Microsoft/BotBuilder-Samples/blame/5be0364be8614b42469a630e8c7343c96364253e/samples/csharp_dotnetcore/06.using-cards/readme.md#L34)